ddl/job_table.go

```diff
@@ -38,8 +38,7 @@ import (
 )
 
 var (
-	addingDDLJobGeneral = "/tidb/ddl/add_ddl_job_general"
-	addingDDLJobReorg   = "/tidb/ddl/add_ddl_job_reorg"
+	addingDDLJobConcurrent = "/tidb/ddl/add_ddl_job_concurrent"
```

Comment:
You could just use `add_ddl_job_general` directly? We don't need to build a new one, do we?